main.go

```diff
@@ -86,15 +101,49 @@ func Execute(ctx context.Context, clientCloudFront cloudfrontclient.CloudFrontCl
 
 			if invalidationDetail != nil {
 				countPaths = countPaths + float64(*invalidationDetail.Invalidation.InvalidationBatch.Paths.Quantity)
+				invalidationPaths = invalidationDetail.Invalidation.InvalidationBatch.Paths.Items
+			}
+		}
+
+		var logGroupName string
+		var logStreamName string
+
+		// fetch log group and log name from distribution tags
+		tags, err := clientCloudFront.ListTagsForResource(ctx, &cloudfront.ListTagsForResourceInput{
+			Resource: distribution.ARN,
+		})
+
+		for _, tag := range tags.Tags.Items {
+			if *tag.Key == InvalidationLogGroupKey {
+				logGroupName = *tag.Value
 			}
+
+			if *tag.Key == InvalidationLogStreamKey {
+				logStreamName = *tag.Value
+			}
+		}
+
+		// verify log group and log stream exist
+		err = logsClient.Verify(ctx, logGroupName, logStreamName)
+		if err != nil {
+			return fmt.Errorf("failed to add logs: %w", err)
+		}
+
+		// send logs to cloudwatch
+		err = logsClient.Flush(ctx, logGroupName, logStreamName, cloudwatchlogstypes.InputLogEvent{
+			Message:   aws.String(fmt.Sprintf("InvalidationRequest: %g, InvalidationPathCounter: %g, InvalidatedPaths: %s", countInvalidations, countPaths, strings.Join(invalidationPaths, ","))),
```

Comment:
My recommendation:

* Make it JSON
* Fields that we can have are: id, total number of paths for that invalidation, list/slice of the paths.